### PR TITLE
Aikidopy uses outdated WorldInteractiveMarker

### PR DESCRIPTION
**Pull request by egordon**
Aikidopy relies on the deprecated WorldInteractiveMarkerViewer which was removed in https://github.com/personalrobotics/aikido/pull/521 

This updates that interface.

Note: this depends on #598 , as this should fix the one failing (not required) test.

***

**Before creating a pull request**

- [x] Document new methods and classes
- [x] Format code with `make format`

**Before merging a pull request**

- [x] Set version target by selecting a milestone on the right side
- [x] Summarize this change in `CHANGELOG.md`
- [x] Add unit test(s) for this change

### CODE REVIEW

File: .ci/script_catkin.sh [hunk 1]
```diff
@@ -20,7 +20,7 @@ fi
 # Build aikidopy and pytest
 if [ $BUILD_AIKIDOPY = ON ]; then
   ./scripts/internal-run.sh catkin build --no-status --no-deps -p 1 -i --cmake-args -DCMAKE_BUILD_TYPE=$BUILD_TYPE -DTREAT_WARNINGS_AS_ERRORS=OFF -DCODECOV=OFF --make-args aikidopy -- aikido
-  ./scripts/internal-run.sh make -C build/aikido pytest
+  $SUDO ./scripts/internal-run.sh make -C build/aikido pytest
```
Comment (sniyaz) [inline .ci/script_catkin.sh:23]: What's the reason for this change? This may be dumb, seems unrelated to the interface change.

File: python/aikidopy/CMakeLists.txt [hunk 2]
```diff
@@ -94,6 +88,12 @@ set_target_properties(aikidopy
     DEBUG_POSTFIX ""
 )
 
+execute_process(COMMAND ${PYTHON_EXECUTABLE} -c
```
Comment (sniyaz) [inline python/aikidopy/CMakeLists.txt:91]: Ditto: What's the reason for this change? This may be dumb, seems unrelated to the interface change.

**Comment by sniyaz** [comment]
Heads up: I changed the base branch to your GH actions one so we can see a better diff.

This seems mostly fine, I just left two comments: I'm a bit confused since those changes don't seem to relate to `WorldInteractiveMarkerViewer -> InteractiveMarkerViewer`

File: python/aikidopy/CMakeLists.txt (hunk 2 above)
Comment (egordon) [reply python/aikidopy/CMakeLists.txt:91]: Just a small readability change, since this variable is only used in the below lines.

File: .ci/script_catkin.sh (hunk 1 above)
Comment (egordon) [reply .ci/script_catkin.sh:23]: CI fails to run without this addition, since the installation step requires sudo.

**Comment by egordon** [comment]
@sniyaz Responded for review comments and updated CHANGELOG. Ready for re-review

**Review by hejia-zhang — APPROVED** [review]
